src/Docfx.Common/Json/System.Text.Json/SystemTextJsonUtility.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+#nullable enable
+
+namespace Docfx.Common;
+
+/// <summary>
+/// Utility class for JSON serialization/deserialization.
+/// </summary>
+public static class SystemTextJsonUtility
+{
+    /// <summary>
+    /// Default JsonSerializerOptions options.
+    /// </summary>
+    public static readonly JsonSerializerOptions DefaultSerializerOptions;
+
+    /// <summary>
+    ///  Default JsonSerializerOptions options with indent setting.
+    /// </summary>
+    public static readonly JsonSerializerOptions IndentedSerializerOptions;
+
+    static SystemTextJsonUtility()
+    {
+        DefaultSerializerOptions = new JsonSerializerOptions()
+        {
+            // DefaultBufferSize = 1024 * 16, // TODO: Set appropriate buffer size based on benchmark.(Default: 16KB)
+            AllowTrailingCommas = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull | JsonIgnoreCondition.WhenWritingDefault,
+            PropertyNameCaseInsensitive = true,
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+            // DictionaryKeyPolicy = JsonNamingPolicy.CamelCase, // This setting is not compatible to `Newtonsoft.Json` serialize result.
+            NumberHandling = JsonNumberHandling.AllowReadingFromString,
+            Converters =
+            {
+                new JsonStringEnumConverter(),
+                new ObjectToInferredTypesConverter(), // Required for `Dictionary<string, object>` type deserialization.
```

Comment:
Can we change `Dictionary<string, object> Others` -> `Dictionary<string, JsonElement> Others` if that is the only place using this converter?